api/src/main/java/org/pmiops/workbench/exceptions/ExceptionUtils.java

```diff
@@ -71,10 +71,6 @@ public static WorkbenchException convertLeonardoException(
     if (isSocketTimeoutException(e.getCause())) {
       throw new GatewayTimeoutException();
     }
-    if (e.getCode() == HttpServletResponse.SC_CONFLICT) {
```

Comment:
This was masking Leo errors. Removing to give us the underlying error.

Reply:
Not as far as I know. I believe that I redundantly added it. In reality this message ends up being handled by the UI:
https://github.com/all-of-us/workbench/blame/main/ui/src/app/components/gke-app-configuration-panels/create-gke-app-button.tsx#L31

https://github.com/all-of-us/workbench/blob/main/ui/src/app/components/apps-panel/expanded-app.tsx#L164